examples/httpGoKit-middleware/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"io"
+	"net/http"
+	"time"
+
+	httptransport "github.com/go-kit/kit/transport/http"
+	"github.com/ricardo-ch/go-tracing"
+)
+
+// this name is use to identify traces inside zipkin
+const (
+	appName = "my-http-server-middleware"
+)
+
+func main() {
+	tracing.SetGlobalTracer(appName, "http://localhost:9411/")
+	defer tracing.FlushCollector()
+
+	svc := stringService{}
+	uppercaseHandler := httptransport.NewServer(
+		makeUppercaseEndpoint(svc),
+		decodeUppercaseRequest,
+		encodeResponse,
+	)
+
+	http.Handle("/uppercase", tracing.HTTPMiddleware("hello-handler", uppercaseHandler))
```

Comment:
it would be more explicit as an example to matchs the endpoint and the operation name like so : 
`tracing.HTTPMiddleware("uppercase-handler", uppercaseHandler)`

Reply:
sure, bad copy paste